practice-one/src/styles/common/step-item.css

```diff
@@ -0,0 +1,15 @@
+.step-title {
```

Comment:
[step-item.css](https://github.com/ronlechiht/html-css-practice/pull/5/files#diff-6f133d2a3f86aea9100365fe918d3c8eea3075fd913272f1749b6906312ffa0a) You can name it step.css